webapp/src/js/controllers/inbox.js

```diff
@@ -713,7 +713,8 @@ var feedback = require('../modules/feedback'),
         $log.error('Application cache update error', err);
       });
       if (
-        $window.applicationCache.status === $window.applicationCache.UPDATEREADY
+        $window.applicationCache.status === $window.applicationCache.UPDATEREADY ||
+        $window.applicationCache.status === $window.applicationCache.UNCACHED
```

Comment:
Unfortunately, this will cause all incognito sessions to trigger the reload prompt ad-infinitum. 
Incognito seems to have an upper limit of caching capabilities and our inbox.js file exceeds the size, so appcache always fails and it triggers the `uncached` event.

I don't think this is necessary for the fix, as code in 3.2 would only take effect once the user has refreshed the page and no longer on webapp 2.18 code.

I'm not sure if this is something intended or not.